webpack.config.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
 		],
 	},
 	mode,
+	target: 'browserslist',
```

Comment:
I assume we will need to apply the same changes in the shared config in `@wordpress/scripts`. I can take care of it later, once the build process for Gutenberg is polished.

Reply:
Yep, at this moment, this doesn't even have the desired effect because https://github.com/webpack/webpack/pull/11754 wasn't released yet. So it's hard to test properly.

Reply:
A month later, `target: 'browserslist'` now works perfectly after a few more webpack 5 bugfix releases. I also added the `target` config to the shared webpack config in `packages/scripts`.